packages/service-provider-core/src/index.ts

```diff
@@ -24,6 +24,7 @@ import { bsonStringifiers } from './printable-bson';
 import ShellAuthOptions from './shell-auth-options';
 import { ConnectionString } from './connection-string';
 export * from './all-transport-types';
+export * from './all-fle-types';
```

Comment:
Will this cause an error for the contexts where we can't import the fle package? Just checking bc previously we had everything FLE in service-provider-server and not core

Reply:
Uh, good point – I think these should both be `export type` (and other lines as well)? I think so long as we depend on it only at build time we should be fine.

Reply:
Yeah, I'm not sure... Inside `all-fle-types` it's already just "types" and I've checked the compiled JavaScript - it's basically an empty file for `all-fle-types.ts`.